iothub/service/src/DeviceGroup.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Microsoft.Azure.Devices
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Shared;
+    using Newtonsoft.Json;
+    /// <summary>
+    /// The data structure represent the DeviceGroup(A device group is made up of devices and modules whose attributes match the query expression)
+    /// </summary>
+    public class DeviceGroup : IETagHolder
+    {
+        /// <summary>
+        /// name
```

Comment:
These properties' doc comments need to be more descriptive.

Reply:
Elsewhere too.

Reply:
Done